### PR TITLE
Acceptable bundle docs

### PR DESCRIPTION
**Pull request by simonbaird**
JIRA: [HACBS-928](https://issues.redhat.com/browse/HACBS-928)

### CODE REVIEW

**Comment by simonbaird** [comment]
As per discussion on slack, there's likely an impact on the CI for the job [here](https://github.com/redhat-appstudio/build-definitions/blob/main/.tekton/push.yaml#L127-L130). Maybe a `make docs-amend` fixes that, but I'm considering other ideas.

File: Makefile [hunk 1]
```diff
@@ -68,67 +72,87 @@ live-test: ## Continuously run tests on changes to any `*.rego` files, `entr` ne
...
-# (Only one generated doc file currently)
-DOCS_ALL=$(DOCS_ADOC)
-
-build-docs: ## Generate documentation. Use this before commit if you modified any rules or annotations
-	@opa inspect --annotations --format json $(POLICY_DIR) > $(DOCS_TMP_JSON)
-	@gomplate -d rules=$(DOCS_TMP_JSON) -f $(DOCSRC)/$$( basename $(DOCS_ADOC) ).tmpl | cat -s > $(DOCS_ADOC)
+ACCEPTABLE_BUNDLES_YAML=$(DATA_DIR)/acceptable_tekton_bundles.yml
+
+# Use gomplate to generate Asciidoc files as input for Antora
+# (For preview and to produce the our local Github pages doc
```
Comment (lcarva) [inline Makefile:111]: "produce ~~the~~ our"

**Review by lcarva — APPROVED** [review]
This is great.

I definitely see the point of adding an "expiry" date to out of date bundles. It should make it easier to generate more accurate docs.

**Comment by simonbaird** [comment]
Preview at https://simonbaird.github.io/ec-policies/ec-policies/acceptable_bundles.html .

**Review by zregvart — APPROVED** [review]

**Pull request merged**